### PR TITLE
Removed unused code

### PR DESCRIPTION
**Pull request by daviddq**
Removed unused code

### CODE REVIEW

File: src/js/classes/app.js [hunk 1]
```diff
@@ -1528,15 +1528,6 @@ export var App = function(name, version) {
     return tagBeforeCursor;
   };
 
-  this.testRunFrom = function(startTestNode) {
-    ipc.send(
-      'testYarnStoryFrom',
```
Comment (blurymind) [inline src/js/classes/app.js:1533]: you can also get rid of https://github.com/YarnSpinnerTool/YarnEditor/blob/f18777ebe8de5c92b48a31cab687cfffaefe9222/electron/main.js#L93

as well as
https://github.com/YarnSpinnerTool/YarnEditor/blob/f18777ebe8de5c92b48a31cab687cfffaefe9222/electron/main.js#L101

and yarnRunnerWindow references in the samefile

Comment (daviddq) [reply src/js/classes/app.js:1533]: I use it as webpage as well, but my narrative designer will use it in Electron.

Code in YarnEditor/electron/* is not _eslinted_. I will keep it unformated. If you want it formated, please, update the "npm run format" script.

Comment (daviddq) [reply src/js/classes/app.js:1533]: I think this is unused as well: 

https://github.com/YarnSpinnerTool/YarnEditor/blob/b5a6bdba6c7f51af5218451f7d5382e3a69bb702/src/js/classes/app.js#L1899

Can you confirm?

Comment (blurymind) [reply src/js/classes/app.js:1533]: wow, that is one long function. I need to investigate it. Its possible it did something useful in the past, but somehow ended up becoming cut off. Yarn has a few legacy things lying around. If nothing is referencing it in the rest  of the app, thats likely the case

Comment (blurymind) [reply src/js/classes/app.js:1567]: Feel free to remove it if you can't think of any use for it. Its completely dead code atm

**Comment by blurymind** [comment]
good work! :+1: 
Thank you for working on this :)
merged now :tada:

**Pull request merged**